package.json

```diff
@@ -50,9 +50,9 @@
     "fork-ts-checker-webpack-plugin": "^1.3.3",
     "gulp": "^4.0.1",
     "gulp-concat": "^2.6.1",
+    "html-webpack-plugin": "4.0.0-beta.5",
```

Comment:
Is this change intended?

Reply:
This probably happened when I installed [web audio typedefs](https://github.com/oppia/oppia/pull/6708#discussion_r287855058) then I removed. I don't think this is intentional.

Reply:
It is okay to keep it ordered this way.

Reply:
It should be the same order as the one in the develop branch.

Reply:
Looks like it's done.